master/reference/public-cloud/gce.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Deploying Calico Kubernetes on GCE
+---
+
+To deploy Calico in [Google Compute Engine][GCE], you must ensure that the
+proper firewall rules have been made and that traffic between containers on
+different hosts is not dropped by the GCE fabric. There are a few different
+options for doing this depending on your deployment.
+
+#### Configure GCE Firewall Rules
+
+Calico requires the following firewall rules to function in GCE.
+
+| Description      | Protocol | Port Range |
+|:-----------------|:---------|:-----------|
+| BGP              | TCP      | 179        |
+| \*IPIP           | 4        | all        |
+
+>\*IPIP: This rule is required only when using Calico with IPIP encapsulation.
+Keep reading for information on when IPIP is required in GCE.
+
+#### Routing Traffic
+
+To use Calico in GCE, either [ipip encapsulation][IPIP] and NAT outgoing must
```

Comment:
I think we should split these up into two subsections of "Routing traffic"

and then we should make the answer to this question more apparent in this section:

> This rule is required only when using Calico with IPIP encapsulation. Keep reading for information on when IPIP is required in GCE